src/main/scala/leon/purescala/ScalaPrinter.scala

```diff
@@ -39,6 +41,13 @@ class ScalaPrinter(opts: PrinterOptions,
       case m @ FiniteMap(els, k, v)  => p"Map[$k,$v]($els)"
 
       case InfiniteIntegerLiteral(v) => p"BigInt($v)"
+      case StringLiteral(v) =>
```

Comment:
Is this any different from the PrettyPrinter version? It doesn't look like it. If it's the same, you don't need to override it here.